server/src/expense/expense.service.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  Injectable,
+  NotFoundException,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { PrismaService } from '../prisma/prisma.service';
+import { CreateExpenseDto, UpdateExpenseDto } from './dto';
+
+@Injectable()
+export class ExpenseService {
+  constructor(private prisma: PrismaService) {}
+
+  async createExpense(userId: number, dto: CreateExpenseDto) {
```

Comment:
You need to validate if the user belongs to the house using userId and houseId!

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/102/commits/1f872e37dae2b6b822a67d341b0f6e8a8ba6e451